spec/classes/init_spec.rb

```diff
@@ -587,6 +587,62 @@
           }.to raise_error(Puppet::Error)
         end
       end
+
+      context "with limits_fragments_hiera_merge parameter specified as a non-boolean or non-string on #{v[:osfamily]} with #{v[:releasetype]} #{v[:release]}" do
+        let :facts do
+          { :osfamily => v[:osfamily],
+            :"#{v[:releasetype]}" => v[:release],
+          }
+        end
+        let (:params) { {:limits_fragments_hiera_merge => ['not_a_boolean', 'not_a_string'] } }
+        it 'should fail' do
+          expect {
+            should contain_class('pam')
+          }.to raise_error(Puppet::Error)
```

Comment:
Can you please place the error message here? A quick way is to change to 

```puppet
}.to raise_error(Puppet::Error,/asdf/)
```

and note the message from the error.